src/commands/runDraftTool/helper/runDraftHelper.ts

```diff
@@ -59,19 +62,22 @@ export async function downloadDraftBinary() {
    if (failed(downloadResult)) {
       return {
          succeeded: false,
-         error: [`Failed to download draft binary: ${downloadResult.error[0]}`]
+         error: `Failed to download draft binary: ${downloadResult.error} from ${draftDownloadUrl}`
       };
    }
    //If linux check -- make chmod 0755
    fs.chmodSync(destinationFile, '0755');
-   return succeeded(downloadResult);
+   return {
+      succeeded: true,
+      result: null
+   };
 }
 
 function getBinaryFileName() {
    let architecture = os.arch();
    let operatingSystem = os.platform().toLocaleLowerCase();
 
-   if (architecture === 'x64') {
+   if (architecture === 'x64' || architecture === 'ia32') {
```

Comment:
💡💡 Curious question: Why the need of `i386` 32 bit support? please investigate, some archs of O/S'es from mid last year have stopped supporting 32 bits from future releases. Thanks.

Reply:
COol, please see discussion here: https://github.com/Azure/aks-devx-tools/pull/80#discussion_r1062740907